modules/tensor_mechanics/test/src/materials/LAROMANCE3TileTest.C

```diff
@@ -0,0 +1,2827 @@
+            {821.0, 905.0}},
+           {{2610896470851.053, 6000000000000.003},
+            {400000000000.0, 600000000000.0},
+            {25.68, 67.12},
+            {0.0, 0.2},
+            {881.0, 965.0}},
+           {{2956225926813.723, 6000000000000.003},
+            {400000000000.0, 600000000000.0},
+            {55.28, 96.72},
+            {0.0, 0.2},
+            {821.0, 905.0}},
+           {{2610896470851.053, 6000000000000.003},
+            {400000000000.0, 600000000000.0},
+            {55.28, 96.72},
+            {0.0, 0.2},
+            {881.0, 965.0}}},
+          {{{2610896470851.053, 6000000000000.003},
+            {399906370400.5274, 600000000000.0},
+            {25.68, 67.12},
+            {0.0, 0.2},
+            {881.0, 965.0}},
+           {{2990889132960.971, 6000000000000.003},
+            {400000000000.0, 600000000000.0},
+            {55.28, 96.72},
+            {0.0, 0.2},
+            {773.0, 845.0}},
+           {{2756786861082.1323, 6000000000000.003},
+            {399667396951.5037, 600000000000.0},
+            {55.28, 96.72},
+            {0.0, 0.2},
+            {821.0, 905.0}}}};
+}
+
+template <bool is_ad>
+std::vector<std::vector<std::vector<std::vector<Real>>>>
+LAROMANCE3TileTestTempl<is_ad>::getInputLimits()
+{
+  return {{{{2968622347857.638, 6000000000000.003},
+            {400000000000.0, 600000000000.0},
+            {25.68, 67.12},
+            {0.0, 0.2},
+            {821.0, 905.0}},
+           {{2610896470851.053, 6000000000000.003},
+            {400000000000.0, 600000000000.0},
+            {25.68, 67.12},
+            {0.0, 0.2},
+            {881.0, 965.0}},
+           {{2956225926813.723, 6000000000000.003},
+            {400000000000.0, 600000000000.0},
+            {55.28, 96.72},
+            {0.0, 0.2},
+            {821.0, 905.0}},
+           {{2610896470851.053, 6000000000000.003},
+            {400000000000.0, 600000000000.0},
+            {55.28, 96.72},
+            {0.0, 0.2},
+            {881.0, 965.0}}},
+          {{{2610896470851.053, 6000000000000.003},
+            {399906370400.5274, 600000000000.0},
+            {25.68, 67.12},
+            {0.0, 0.2},
+            {881.0, 965.0}},
+           {{2756786861082.1323, 6000000000000.003},
+            {399667396951.5037, 600000000000.0},
+            {55.28, 96.72},
+            {0.0, 0.2},
+            {821.0, 905.0}},
+           {{2403341139508.535, 6000000000000.003},
+            {399232564175.43, 600000000000.0},
+            {55.28, 96.72},
+            {0.0, 0.2},
+            {881.0, 965.0}}}};
+}
+
+template <bool is_ad>
+std::vector<std::vector<std::vector<std::vector<Real>>>>
+LAROMANCE3TileTestTempl<is_ad>::getCoefs()
+{
+  return {{{{1.}, {1.}, {1.}}, {{1.}, {1.}, {1.}}, {{1.}, {1.}, {1.}}, {{1.}, {1.}, {1.}}},
+          {{{1.}, {1.}, {1.}}, {{1.}, {1.}, {1.}}, {{1.}, {1.}, {1.}}}};
+}
+
+template <bool is_ad>
+std::vector<std::vector<Real>>
+LAROMANCE3TileTestTempl<is_ad>::getClassificationLuu()
```

Comment:
Do you need such a complicated paritition info?

Reply:
in this region, also partitions overlap, so we need the calibrated GPR model with all these coefficients.

Reply:
Still a bit confused why it has to be a calibrated GPR and not a simple one, much like the ROM coefs?

Reply:
For example, why can't you have just a simple linear partition weight fit?

Reply:
Because we do not know the partition weight a priori for a given input. There is no direct relation at the ready. We do, however, have a Gaussian Process Regression model which has been calibrated with VPSC simulation data. The calibrated GPR model is contained in the arrays that are within the material file (in this case in the test-model), which provides the mean of a Gaussian probability distribution of functions relating the classification of a partition to the input domain. But it does not provide some direct, sampled relation. For a given input, we need to re-evaluate the probability distribution of functions again, considering the new input, as well as all inputs upon which the GPR has been calibrated (which is all contained in the GPR-related arrays of the material model). In short, we need the full GPR model, and not some reduced version of it. Here is a resource to get a better understanding if anyone is interested: http://gaussianprocess.org/gpml/chapters/RW.pdf